versioned_docs/version-8.7/self-managed/identity/user-guide/configuration/alternative-db.md

```diff
@@ -0,0 +1,200 @@
+---
+id: alternative-db
+title: "Using Different DBs with Identity for OIDC"
+sidebar_label: "Different DBs with Identity for OIDC"
+description: "Configure Identity to use other DBs with OIDC"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Overview
+
+This guide is intended for customers those internal policies or compliance requirements prevent the use of PostgreSQL.
+
+## Scope
+
+This documentation focuses solely on components of Camunda 8 that can operate without PostgreSQL. Components such as Optimize and Web Modeler, which require PostgreSQL for proper functionality, are excluded from this guide and should be disabled in environments where PostgreSQL is not used.
+
+```yaml
+postgresql:
+  enabled: false
+identityKeycloak:
+  enabled: false
```

Comment:
:question: Isn't the choice to use Keycloak here irrelevant? I understand it's relevant for BofA in particular, but not that much for this documentation, no?

Reply:
e43b0188f